options.go

```diff
@@ -82,10 +86,9 @@ func WithEndpointErrorHandler(eh EndpointErrorHandler) EndpointOption {
 
 type withUpdatePeriod struct{ updatePeriod time.Duration }
 
```

Comment:
warning: exported function WithUpdatePeriod should have comment or be unexported (golint)